multi_import/importer.py

```diff
@@ -398,6 +447,48 @@ def process_row(self, row, data, context):
 
         data.processed = True
 
+    def process_row_subsequent_pass(self, row, data, context, serializer_class):
+        if row.errors:
+            return
+
+        if not data.instance:
+            return
+
+        serializer = serializer_class(
+            instance=data.instance,
+            data=row.data.copy(),
+            context=context,
+            partial=data.instance is not None,
+        )
+
+        if not serializers.might_have_changes(serializer):
```

Comment:
this line reads really weird... the `might_have_changes` part is really throwing me off-- might have and might not have feels like the same thing (glass half full vs glass half empty kinda situation).

Reply:
I wasn't happy with any of the alternative method names, that I have thought of so far.

So re-wrote the code like this:

```python
does_not_have_changes = not serializers.might_have_changes(serializer)

if does_not_have_changes:
```